ocs_ci/ocs/ui/page_objects/s3_vector_tab.py

```
@@ -0,0 +1,443 @@
+        logger.info("Clicking Create bucket button")
+        self.do_click(self.bucket_tab["create_bucket_button"], enable_screenshot=True)
+
+        logger.info("Selecting 'Create via Object Bucket Claim'")
+        self.do_click(
+            self.bucket_tab["create_bucket_button_obc"], enable_screenshot=True
+        )
+
+        logger.info(f"Entering bucket name: {bucket_name}")
+        self.do_send_keys(self.bucket_tab["obc_bucket_name_input"], bucket_name)
+
+        logger.info(f"Selecting storage class: {storageclass}")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)